### PR TITLE
added ui

### PR DESCRIPTION
**Pull request by AtillaTahak**
Create a main.rb file that will serve as your console app entry-point. It should have methods that do the following:
List all books.
List all people.
Created a person (teacher or student, not a plain Person).
Create a book.
Create a rental.
List all rentals for a given person id.
In main.rb define the entry point, this will be a method called main that is invoked at the end of your file. -
This method does the following:
Present the user with a list of options to perform.
Lets users choose an option.
Have a way to quit the app.

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  else
+    puts 'Invalid entry!'
+    init
+  end
+  student = Student.new(age, name)
+  student.parent_permission = permission
+  student.type = 'Student'
+  add_person(student)
+  success('Person')
+end
```
Comment (Emmyn5600) [inline main.rb]: - [ ] I have an error while creating the person, please fix it.

image

File: main.rb [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  person_index = gets.chomp.to_i
+  print 'Date (yyyy/mm/dd): '
+  date = gets.chomp
+  person = @person_arr[person_index]
+  book = @book_arr[book_index]
+  Rental.new(date, person, book)
+  success('Rental')
+  init
+end
+
```
Comment (Emmyn5600) [inline main.rb]: - [ ] I had an issue too while creating the rentals.

image

**Review by Emmyn5600 — CHANGES_REQUESTED** [review]
Hi @AtillaTahak 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight :champagne:

- All linters are passing :heavy_check_mark:
- Good commit history :heavy_check_mark:
- Good PR description :heavy_check_mark:

## Required Changes :recycle:

- [ ] Check the comment under the review

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!:clap::clap::clap:

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 2 above)
Comment (AtillaTahak) [reply main.rb]: You have to create person and book for creating new rentals

**Review by t-yanick — APPROVED** [review]
Hi @AtillaTahak, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**